Readme.md

```diff
@@ -546,6 +546,8 @@ subcommand is specified ([example](./examples/defaultCommand.js)).
 
 You can add alternative names for a command with `.alias()`. ([example](./examples/alias.js))
 
+Commands added with `.command()` automatically inherit settings for which inheritance is meaningful from the parent command, but only upon the subcommand creation. The setting changes made after calling `.command()` are not inherited.
```

Comment:
I have struggled with whether and how to describe this in the past so thanks for having a go. I like the direction here, fairly high level but clarifies the implicit ordering. Maybe:

> Commands created with `.command()` automatically inherit relevant settings from the parent command. This is done during creation, and any later setting changes to the parent are not inherited.

Reply:
Not sure I like "relevant", seems a little vague. Maybe simplify what I suggested to "for which that is meaningful”?

I like the rest though 👍

Reply:
Ha, thanks, I was a little dubious about the `relevant` too. Given your reaction, I think leave it out, just "inherited settings". It isn't the whole story, but any attempt to hint at more just leads to questions about what is being hinted at without actually helping. (e.g. "some", "relevant", "for which inheritance is meaningful")

Reply:
"inherit inherited settings" just doesn't sound good because of the repetition.

Reply:
> Commands created with .command() automatically inherit settings from the parent command. This is done during creation, and any later setting changes to the parent are not inherited.

Reply:
Now it makes it seem like all settings are inherited 🤷‍♂️

Reply:
Mind dump, not condensed!

I wondered about "all" and initially thought it would be obvious that not all settings are copied. But I guess some users might carefully set settings after adding the commands to avoid them being inappropriately copied. And I would hate to cause that.

Words: "relevant", "some", "selected", "appropriate"

In the working model in my mind there are "inherited settings" which are copied, with a somewhat circular definition of what that means. I'll try writing something out...

"automatically copy inherited settings from the parent command"

Hmm, maybe! That fits in with my longterm working model, matches the subsequent mention of `copyInheritedSettings` in the next paragraph, which provides a clue for people to find out the actual code. (And on that note, I like what I put in the JSDoc for that routine.)

Reply:
Going for "`.command()` automatically copies the inherited settings" just like in the next paragraph, how do you like it?

```suggestion
`.command()` automatically copies the inherited settings from the parent command to the newly created subcommand. This is only done during creation, and any later setting changes to the parent are not inherited.
```

Reply:
LGTM

Reply:
Resolved in https://github.com/tj/commander.js/pull/1917/commits/dfe2fc74a53db43153f4305ec5f9f1cc6a13c64f.